pkg/kritis/secrets/pgpkey.go

```diff
@@ -89,6 +89,14 @@ func parsePrivateKey(privateKey string) (*packet.PrivateKey, error) {
 	if !ok {
 		return nil, fmt.Errorf("Not a private Key")
 	}
+	if passphrase != "" {
+		// Decrypt the private key with the passphrase
+		pb := []byte(passphrase)
+		err := key.Decrypt(pb)
+		if err != nil {
+			return nil, fmt.Errorf("Failed to decrypt private key with passphrase")
```

Comment:
Don't hide the original error. Use something like:

`return fmt.Errorf("Decrypt failed: %v", err)`